micada/pawar2018
# Localize events archive

# What does this pull request do?
Localize events archive

# How should the reviewer test this pull request?
- Update home pages with values for new "Event slug" custom field ("/events" for English and "/es/eventos" for Español)
- Make sure Events are activated under Languages > Settings > Custom post types and Taxonomies
![screen shot 2017-06-24 at 9 13 49 pm](https://user-images.githubusercontent.com/772939/27513187-354490b4-5922-11e7-8feb-5ca9dd823739.png)
- Add translations for the strings on the Events page and the URL slug
![screen shot 2017-06-24 at 9 16 32 pm](https://user-images.githubusercontent.com/772939/27513194-749339c8-5922-11e7-8253-270814d35960.png)

# Include the Trello card for this PR, if there is one.
https://trello.com/c/sPFkemmw
